src/mds/MDSMap.cc

```diff
@@ -635,6 +637,24 @@ void MDSMap::encode(bufferlist& bl, uint64_t features) const
   ENCODE_FINISH(bl);
 }
 
+void MDSMap::sanitize(std::function<bool(int pool)> pool_exists)
+{
+  /* Before we did stricter checking, it was possible to remove a data pool
+   * without also deleting it from the MDSMap. Check for that here after
+   * decoding the data pools.
+   */
+
+  for (auto it = data_pools.begin(); it != data_pools.end();) {
+    dout(0) << "pool " << *it << dendl;
```

Comment:
This part isn't an error so probably was a debugging print, or meant to be at a lower severity?

(also I generally tend to use derr instead of dout(0))

Reply:
Ya, that was debugging left in. Thanks!